azure_functions_worker/dispatcher.py

```diff
@@ -484,6 +486,11 @@ async def _handle__function_environment_reload_request(self, req):
                 self._create_sync_call_tp(self._get_sync_tp_max_workers())
             )
 
+            # Apply PYTHON_ENABLE_DEBUG_LOGGING
+            if is_envvar_true(PYTHON_ENABLE_DEBUG_LOGGING):
+                root_logger = logging.getLogger()
+                root_logger.setLevel(logging.DEBUG)
+
```

Comment:
The comment doesn't add much help. You can remove this.

Reply:
will do